### PR TITLE
Device: Deltanet Lambdatrail LT2-S,HS

### PR DESCRIPTION
**Pull request by Bierchermuesli**
Deltanet Lambdatrail integration, optical equipment. This model is a simple 10GE/FC Media converter. 

image

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: mibs/deltanet/deltanet.mib [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Deltanet-MIB DEFINITIONS ::= BEGIN
```
Comment (PipoCanaja) [inline mibs/deltanet/deltanet.mib:1]: Hi @Bierchermuesli 
Could you please rename the file to `Deltanet-MIB`. We need to have all MIBs named after their official definition for readability and deduplication ... 
Thanx

File: mibs/deltanet/lambdatrail.mib [hunk 2]
```diff
@@ -0,0 +1,1147 @@
+LAMBDATRAIL-MIB DEFINITIONS ::= BEGIN
```
Comment (PipoCanaja) [inline mibs/deltanet/lambdatrail.mib:1]: Same here with `LAMBDATRAIL-MIB `

File: mibs/deltanet/lambdatrail2-s.mib [hunk 3]
```diff
@@ -0,0 +1,1143 @@
+Lambdatrail2s-MIB DEFINITIONS ::= BEGIN
```
Comment (PipoCanaja) [inline mibs/deltanet/lambdatrail2-s.mib:1]: And same here with `Lambdatrail2s-MIB`

**Review by PipoCanaja — CHANGES_REQUESTED** [review]
Hi @Bierchermuesli 
Thanx for your contribution. Could you please provide the complete test data (snmprec file including all OIDs, and JSON file computed out of it) to feed non regression tests. And also the MIB files must be renamed with their official `DEFINITION`. 
Thanx

File: includes/definitions/discovery/lambdatrail.yaml [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+                    index: 'portClientLaserTemp.{{ $index }}'
+                    skip_values:
+                        -
+                            oid: portClientLaserTemp
+                            op: '='
+                            value: -1
+                -
+                    oid: systemTemperature
+                    value: systemTemperature
+                    num_oid: '.1.3.6.1.4.1.35616.2.1.1.1.0'
```
Comment (PipoCanaja) [inline includes/definitions/discovery/lambdatrail.yaml:54]: You need to replace this line with: 
`num_oid: '.1.3.6.1.4.1.35616.2.1.1.1.{{ $index }}'`
There is only one entry so it won't change your display, but this ensures that the line is compatible with a table.

Comment (Bierchermuesli) [reply includes/definitions/discovery/lambdatrail.yaml]: Thanks for your instruction! 

I hope I fixed the nessessary stuff

Please advise for:
- lambdatrail is not realy an os, its more a product family, is this the right way?
- lt2-s and hs variante don't have much in common -  better have seperate definition/discovery.yaml ?
- the dbm low/high_limit don't have a divisor - should I switch to a php disovery?
- how can i set default low/high values for e.g. Temp (not delivered from snmp)

Comment (Bierchermuesli) [reply includes/definitions/discovery/lambdatrail.yaml]: @PipoCanaja anything I can optimize here?

**Comment by Bierchermuesli** [comment]
@PipoCanaja anything I have to do here? please advice

**Comment by Jellyfrog** [comment]
Youre trying to merge a bunch of commits here thats not yours, you need to fix that

**Comment by Bierchermuesli** [comment]
how could this happend? I see 24 commits of mine in this pull? can you elaborate this a biit?

**Review by murrant — APPROVED** [review]

**Pull request merged**